hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/functional/TestCleanActionExecutor.java

```diff
@@ -138,7 +138,7 @@ void testPartialCleanFailure(CleanFailureType failureType) throws IOException {
     when(activeTimeline.getCleanerTimeline()).thenReturn(cleanTimeline);
     when(cleanTimeline.getInstants()).thenReturn(Collections.singletonList(cleanInstant));
     when(activeTimeline.readCleanerPlan(cleanInstant)).thenReturn(cleanerPlan);
-    when(activeTimeline.readCleanerInfoAsBytes(cleanInstant)).thenReturn(TimelineMetadataUtils.serializeCleanerPlan(cleanerPlan));
+    when(activeTimeline.readCleanerInfoAsBytes(cleanInstant)).thenReturn(Option.of(convertMetadataToBytArray(cleanerPlan)));
```

Comment:
Is this method `activeTimeline.readCleanerInfoAsBytes` still used?  Let's create a follow-up JIRA ticket to clean up the usage on byte array related handling in the `HoodieTimeline` and subclasses.